### PR TITLE
Add population to admins

### PR DESCRIPTION
**Pull request by DavidKarlas**

### CODE REVIEW

**Review by sasastanojkov — APPROVED** [review]

File: schema/admins/locality.yaml [hunk 1]
```diff
@@ -36,3 +36,11 @@ properties:
...
         description: Context entity is the most granular entity that logically contains given entity (but doesn't have to contain it spatially due to minor discrepancies in geometries)
         type: string
+      population:
+        description: Number of citizen in locality.
+        type: integer
+      populationYear:
+        description: Year when population was counted.
+        type: integer
+        minimum: 1900
+        maximum: 2200
```
[the remark this answers is not in the document]
Comment (vcschapp) [reply schema/admins/locality.yaml:46]: > Should this be restricted that if the year is supplied then the population must also be?

Yes.

**Comment by vcschapp** [comment]
Two thoughts:

1. I'd personally like to solve political perspectives before solving this since PP is a fundamental problem.
2. Related, which population is this? Or, population counted according to which PP?

File: schema/admins/locality.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (DavidKarlas) [reply schema/admins/locality.yaml]: I asked around, and looks like nobody is actually using Population Year... Hence deleted it from this PR...

**Comment by jwass** [comment]
Where do we stand with @vcschapp's comment about waiting until political views are sorted out?

If we assume that since different political views will have different entities, a population field won't interfere with this because we can have a per-political view population. In other words, since we don't know the exact direction of political views, this will likely be consistent with that approach.

**Review by vcschapp — COMMENTED** [review]
One other question I have is, what is the source of this data?

Are we echoing it from OSM, or taking it from elsewhere?

I think really at some level, my question is digging at where the boundary "a map" or geospatial data lies. Is the population within it?

**Comment by DavidKarlas** [comment]
Population feels pretty critical information for rendering maps in sense it is major indicator on which LOD name of city should be drawn. Not sure if I'm answering question, but ye OSM is only source for that at this moment....

**Review by MarkLebegge-TomTom — APPROVED** [review]

File: schema/admins/locality.yaml [hunk 2]
```diff
@@ -36,3 +36,6 @@ properties:
       contextId:
         description: Context entity is the most granular entity that logically contains given entity (but doesn't have to contain it spatially due to minor discrepancies in geometries)
         type: string
+      population:
+        description: Number of citizen in locality.
```
Comment (jwass) [inline schema/admins/locality.yaml:40]: nit: "Number of citizen**s**".

You could just say "population in the locality" or something like that too.

**Review by jwass — APPROVED** [review]

File: schema/admins/locality.yaml (hunk 2 above)
Comment (DavidKarlas) [reply schema/admins/locality.yaml]: Done.

**Review by vcschapp — COMMENTED** [review]
Let's merge it for the time being.

**Review by vcschapp — APPROVED** [review]

**Pull request merged**